conf/messages

```diff
@@ -36,7 +36,6 @@ estimated_tax_liability.eoyEstimate.p2                          = It''s based on
 estimated_tax_liability.inYearEstimate.heading                  = Current estimate: {0}
 estimated_tax_liability.inYearEstimate.p1                       = This is an estimate of the tax you owe from 6 April {0} to {1}.
 estimated_tax_liability.inYearEstimate.p2                       = It''s based on the information you provide us in your quarterly reports.
```

Comment:
This line's changed in the prototype to "It's based on the information you report through software."

Reply:
Well spotted, thanks :+1: